graph.go

```diff
@@ -195,16 +195,21 @@ func (c *GraphCmd) Execute(args []string) error {
 }
 
 func relPath(base, path string) string {
-	rp, err := filepath.Rel(base, path)
+	pathWithoutSymlinks, err := filepath.EvalSymlinks(path)
```

Comment:
Why do you need to evaluate symlinks here? Shouldn't srclib-go be agnostic of whether the path to the repo is symlinked to another location?